entrypoint.sh

```
@@ -1,10 +1,14 @@
 #!/bin/sh
 
+# Collect any extra flags passed as input
+EXTRA_ARGS="$INPUT_ARGS"
+
 if [ "$LOG_LEVEL" = "debug" ]; then
   echo "DEBUG MODE: Skipping setting GitHub output."
   # Simply run vnext to print debug output to STDOUT
-  vnext
+  echo "Running: vnext $EXTRA_ARGS"
+  vnext $EXTRA_ARGS
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)